pulsar-broker/src/main/java/org/apache/pulsar/PulsarStandalone.java

```diff
@@ -434,10 +438,13 @@ public void close() {
         }
     }
 
-
-    private void startBookieWithRocksDB() throws Exception {
-        log.info("Starting BK with RocksDb metadata store");
-        String metadataStoreUrl = "rocksdb://" + Paths.get(metadataDir).toAbsolutePath();
+    private void startBookieWithMetadataStore() throws Exception {
+        if(StringUtils.isBlank(metadataStoreUrl)){
+            log.info("Starting BK with RocksDb metadata store");
+            metadataStoreUrl = "rocksdb://" + Paths.get(metadataDir).toAbsolutePath();
+        }else{
+            log.info("Starting BK with metadata store:",metadataStoreUrl);
+        }
```

Comment:
Please format this code block according to our code style settings. Briefly, add necessary blanks.

Reply:
Thanks for your review and i'm busy for otherthing, pick it up in  the next week maybe.